apps/ui/components/reservation/ReservationCancellation.tsx

```diff
@@ -198,19 +235,20 @@ const ReservationCancellation = ({ id }: Props): JSX.Element => {
     return <Spinner />;
   }
 
-  const reservationUnit = reservation.reservationUnits[0];
+  const reservationUnit =
+    reservation.reservationUnits && reservation.reservationUnits[0];
```

Comment:
This might be cause a Typescript error with TS 5.2 because `reservationUnit: boolean | ReservationType` => boolean is not assignable to ReservationType down the line. `Ternary` or `?.` is preferable so it's `ReservationType | undefined`

Reply:
Yeah, I ended up using `reservation?.reservationUnits?.[0]` here too. 👍